docs/release-guide.md

```diff
@@ -3,6 +3,43 @@ id: release-guide
 title: Release Guide
 ---
 
+## Release Flow
+
+The release flow of Apache APISIX follows [GitLab flow](https://docs.gitlab.com/ee/topics/gitlab_flow.html), instead of Git flow or Github flow. Furthermore, [Release branches with GitLab flow](https://docs.gitlab.com/ee/topics/gitlab_flow.html#release-branches-with-gitlab-flow) is the preferred way. The chart below will take the release `v2.3` of [apache/apisix-dashboard](https://github.com/apache/apisix-dashboard) as an example to illustrate the details.
+
+![Release Flow](/img/release_flow.png)
```

Comment:
broken?

Reply:
@juzhiyuan worked on my local. I've uploaded the png file under `/static/img/` directory within this PR. I noticed the links of other images are all like `static.apiseven.com/xxx`. Need I also upload to `static.apiseven.com` beforehand?

Reply:
🤔 Links start with `static.apiseven.com` are from blogs.

fine, I know what happened, once we merged and deployed, it will show correctly。

Reply:
@juzhiyuan  OK, I know. Here is the screenshot, for your information.

![image](https://user-images.githubusercontent.com/2542401/104813886-ddf97400-5846-11eb-917c-ab0e300a4c70.png)